compiler/ast/ast_build.nim

```diff
@@ -0,0 +1,80 @@
+## This module provides simplified procedures for the AST construction
+
+import ast, lineinfos
+
+proc newIf*(cond, body: PNode, orElse: PNode = nil): PNode =
```

Comment:
Hmm, this brings up an interesting question, do we have more expressions or statements, and also what should we favor?